cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1608,11 +1608,10 @@ if (testing)
 
 endif()
 
-#---Report non implemented options---------------------------------------------------
+#---Report removed options---------------------------------------------------
 foreach(opt afs glite sapdb srp)
   if(${opt})
-    message(STATUS ">>> Option '${opt}' not implemented yet! Signal your urgency to pere.mato@cern.ch")
-    set(${opt} OFF CACHE BOOL "Disabled because not implemented yet (${${opt}_description})" FORCE)
+    message(FATAL_ERROR ">>> Option '${opt}' has been removed in ROOT v6.16.")
```

Comment:
Hi @Axel-Naumann, could you please remove these options from `-Dall=ON` too? It looks like they are still enabled, prompting failures in the CI.